src/poppler/image.py

```diff
@@ -47,8 +46,8 @@ def bytes_per_row(self):
     def const_data(self):
         return self._image.data()
 
-    def copy(self, rect=None):
-        image = self._image.copy(rect or Rectangle()._rect)
+    def copy(self):
```

Comment:
how sure are we that no consumer used the `rect` argument (even if never implemented by the underlying library)? 

If we're not sure, we'll have to keep the argument around (emitting a warning when it's non-`None`, maybe?)